### PR TITLE
schedule : clean up the unused store ID in operator limit

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve? 

clean up the unused store ID in operator limit

- [x] add test 
### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by lhy1024** [comment]
PTAL @rleungx @disksing

File: server/cluster_test.go [hunk 1]
```diff
@@ -339,59 +339,70 @@ func (s *baseCluster) resetStoreState(c *C, storeID uint64, state metapb.StoreSt
...
 }
 
-func (s *baseCluster) testRemoveStore(c *C, clusterID uint64, store *metapb.Store) {
+func (s *baseCluster) testStateAndLimit(c *C, clusterID uint64, store *metapb.Store, beforeState metapb.StoreState, run func(*RaftCluster) error, expectStates ...metapb.StoreState) {
+	// prepare
+	storeID := store.GetId()
 	cluster := s.getRaftCluster(c)
+	cluster.coordinator.opController.SetAllStoresLimit(1.0)
+	s.resetStoreState(c, store.GetId(), beforeState)
+	_, beforeLimit := cluster.coordinator.opController.GetAllStoresLimit()[storeID]
```
Comment (rleungx) [inline server/cluster_test.go]: Since it is a bool variable, the name is a little bit wired.

Comment (lhy1024) [reply server/cluster_test.go]: how about `isOKBefore`?

**Comment by lhy1024** [comment]
PTAL @rleungx @disksing

File: server/cluster.go [hunk 2]
```diff
@@ -859,6 +859,7 @@ func (c *RaftCluster) BuryStore(storeID uint64, force bool) error { // revive:di
 
 	// Bury a tombstone store should be OK, nothing to do.
 	if store.IsTombstone() {
+		c.coordinator.opController.RemoveStoreLimit(store.GetID())
```
Comment (disksing) [inline server/cluster.go]: Is it redundant?

**Comment by lhy1024** [comment]
@nolouch @rleungx PTAL

**Comment by lhy1024** [comment]
@nolouch @Luffbee @shafreeck PTAL

**Pull request merged**